src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -421,13 +500,28 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
    * @memberof MgtPeoplePicker
    */
   protected renderPersonResult(person: IDynamicPerson): TemplateResult {
+    let secondLine;
+
+    const user = person as User;
+    const group = person as Group;
+    if (user.jobTitle) {
+      secondLine = user.jobTitle;
+    } else if (group.mail) {
+      secondLine = group.mail;
+    }
+
+    const classes = {
+      'people-person-job-title': true,
+      uppercase: !!user.jobTitle
```

Comment:
This is only setting the uppercase class if the second line is the job title. Otherwise (such as a group), it is not setting the class so the email is not capitalized